pages/1.11/release-notes/1.11.1/index.md

```diff
@@ -0,0 +1,111 @@
+
+[enterprise]
+### Security
+[/enterprise]
+- Secrets Management Service
+  - Binary Secret files are now supported
+  - Hierarchical access control is now supported.
+
+### Monitoring
+- The DC/OS metrics component now produces metrics in [Prometheus](https://prometheus.io/docs/instrumenting/exposition_formats/) format. [View the documentation](/1.11/metrics).
+- Unified logging API that provides simple access to container (task) as well as system component logs. [View the documentation](/1.11/monitoring/logging/logging-api/logging-v2/).
+
+### Storage
+- DC/OS Storage Service 0.1 (beta) - DSS users will be able to dynamically create volumes based upon profiles or policies to fine-tune their applications' storage requirements. This feature leverages the industry-standard Container Storage Interface (CSI) to streamline the development of storage features in DC/OS by Mesosphere and our community and partner ecosystems. [View the documentation](https://docs.mesosphere.com/services/beta-storage/0.1.0-beta/).[beta type="inline" size="small" /][enterprise type="inline" size="small" /]
+- Pods now support persistent volumes. [View the documentation](/1.11/deploying-services/pods).[beta type="inline" size="small" /]
+
+**Note:** Because these storage features are beta in 1.11, they must be explicitly enabled. Beta features are not recommended for production usage, but are a good indication of the direction the project is headed.
+
+### Updated DC/OS Data Services
+- TLS encryption for DC/OS Kafka, DC/OS Cassandra, DC/OS Elastic, and DC/OS HDFS is now supported.
+- Fault domain awareness for DC/OS Kafka, DC/OS Cassandra, DC/OS Elastic and DC/OS HDFS. Use fault domain awareness to make your services highly available and to allow for increased capacity when needed.
+- New API endpoint to pause a node for DC/OS Kafka, DC/OS Cassandra, DC/OS Elastic, and DC/OS HDFS. Use this endpoint to relaunch a node in an idle command state for debugging purposes.
+- New DC/OS Kafka ZooKeeper service. [View the documentation](/services/kafka-zookeeper).
+- You can now select a DC/OS data service version from a dropdown menu in the DC/OS UI.
+- Improved scalability for all DC/OS data services.
+
+## <a name="issues-fixed"></a>Improvements and Issues Fixed in DC/OS 1.11.1 Release
+
+- INFINITY-3331 - Fixed cleaning up other framework's volumes.
+- DCOS_OSS-2292 - Fixed crashes on empty lines for DC/OS CLI.
+- DCOS_OSS-2255 - Fixed dcos-installer-ui package failure to build.
+- DCOS_OSS-2247 - Fixed fatal error on checks that exceeded timeout on dcos-diagnostic service.
+- DCOS_OSS-2223 - Added encoders to support package registry. Fixed comsos to initialize empty objects for optional json elements from json schema.
```

Comment:
I have a hard time understanding this. @takirala can you improve this?

Reply:
`DCOS_OSS-2223` has nothing to do with package registry. I would rephrase as below:

DCOS_OSS-2223 - Improved the mechanism of rendering service options from package config schema to stop initialization of empty json objects when they are mentioned as optional in schema.

Let me know if that is not descriptive enough / cc: @anithamuthu79

Reply:
Yes and No. The UI still need to make the same changes as cosmos in order to enable the user to take advantage of this feature. But if the user is smart enough, UI can be bypassed and CLI can be used. 

That being said, I actually just realized `DCOS_OSS-2223` has not yet made in to `1.11.1`. `DCOS_OSS-2223` is merged in to cosmos but never went to DC/OS. This should not be here. You can ignore my comments and just delete the release note for `DCOS_OSS-2223`.

Reply:
Removed DCOS_OSS-2223.